common/test/unit/test_stats_service/test_stats_manager.py

```diff
@@ -124,9 +128,71 @@ def test_default_configs(self):
       "all": 4,
       "4xx": 2,
       "5xx": 1,
+      "latency": 0,
       "by_app": {
-        "guestbook": {"all": 2, "4xx": 1, "5xx": 0},
-        "ghostbook": {"all": 2, "4xx": 1, "5xx": 1}
+        "guestbook": {"all": 2, "4xx": 1, "5xx": 0, "latency": 0},
+        "ghostbook": {"all": 2, "4xx": 1, "5xx": 1, "latency": 0}
+      }
+    })
+
+
+class TestDefaultCumulativeSummarizers(unittest.TestCase):
+
+  @classmethod
+  def setUpClass(cls):
```

Comment:
It's better to keep every test independent from each-other. `setUpClass` and `tearDownClass` should be used when you know exactly that there's something common for all tests in the test case. Otherwise it'd be clearer to use just `setUp` and `tearDown`.